src/localization/pt_br/crudbooster.php

```diff
@@ -0,0 +1,161 @@
+    'export_dialog_show_advanced'    => 'Mostrar exportação avançada', 
+    'export_dialog_page_size'        => 'Tamanho da página', 
+    'export_dialog_set_default'      => 'Definir como tamanho de papel predifinido', 
+    'export_dialog_page_orientation' => 'Orientação da página', 
+    'import_page_title'         => 'Importar dados de :module', 
+    'dashboard_default_text'    => 'Bem-vindo ao Painel de Controle de :appname', 
+    'form_back_to_list'         => 'Voltar para a lista de dados de :module', 
+    'not_logged_in'             => 'Você não está logado!', 
+    'menu_navigation'           => "Navegação", 
+    'table_data_not_found'      => "Não há dados disponiveis", 
+    "notice_delete_file_upload" => "* Se você quiser carregar outro arquivo, primeiro excluia o arquivo.", 
+    "file_broken"               => "Oops, parece que o arquivo estava quebrado. Clique em excluir e tente novamente.", 
+    'denied_access' => 'Desculpe, você não tem privilégios para acessar esta área!', 
+    'denied_update' => 'Desculpe, você não tem privilégio para atualizar os dados!', 
+    'denied_create' => 'Desculpe, você não tem privilégio para criar os dados!', 
+    'denied_delete' => 'Desculpe, você não tem privilégio para excluir os dados!', 
+ 
+    'user_image'           => 'Imagem do usuário', 
+    'online'               => 'Online', 
+    'controller_route_404' => 'Controle / Rota não encontra]', 
+    'home'                 => 'Inicio', 
+ 
+    'Privileges_Roles'    => 'Privilégios', 
+    'Add_New_Privilege'   => 'Adicionar novo Privilégio', 
+    'List_Privilege'      => 'Listar Privilégios', 
+    'add_user'            => 'Adicionar Usuário', 
+    'List_users'          => 'Listar Uusário', 
+    'Menu_Management'     => 'Gestão de menus', 
+    'Add_New_Setting'     => 'Adicionar Nova Configuração', 
+    'Add_New_Module'      => 'Adicionar Novo Modulo', 
+    'List_Module'         => 'Listar Módulos', 
+    'Add_New_Statistic'   => 'Adicionar nova estatistica', 
+    'List_Statistic'      => 'Listas estatisticas', 
+    'Add_New_API'         => 'Adicionar nova API', 
+    'API_Generator'       => 'Gerador de API', 
+    'Statistic_Builder'   => 'Construtor de Estatisticas', 
+    'Module_Generator'    => 'Módulo Gerador', 
+    'settings'            => 'Configurações', 
+    'Users_Management'    => 'Gestão de usuários', 
+    'list_API'            => 'Listar API (Documentação)', 
+    'Generate_Screet_Key' => 'Gerar nova chave secreta', 
+    'Add_New_Email'       => 'Adicionar novo e-mail', 
+    'List_Email_Template' => 'Listar modelos de e-mail', 
+    'Email_Templates'     => 'Modelos de e-mail', 
+    'Log_User_Access'     => 'Histórico de acesso', 
+    'SUPERADMIN'          => 'SUPER-ADMNISTRADOR', 
+
+    'click_here'                => 'Clique aqui',
+    'notificationYouHave'       => 'você tem',
+    'notificationNotifications' => 'Notificações',
+
+    'right' => 'direita',
```

Comment:
please notes : this is for html direction to use in html and css classes like : 
`pull-left , pull-right , text-right ...`

to make it easy to use the the view for rtl langs .
unless your language use `right-to-left direction` , you should keep it as : 
```
right => right
left => left
```

thank you .